app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -37,12 +39,17 @@
     private String collectionEventId;
     private StudentImageCollectionEventDao studentImageCollectionEventDao;
     private int imagesProcessed;
+    private Mat imgOverlay;
+
 
     // Image collection parameters
     private static final boolean diagnoseMode = true;
     private static final long timerDiff = 100;
     private static final int numberOfImages = 20;
 
+    // Mat objects for overlay
+    private Mat imgMask  = new Mat();
+    private Mat imgInvMask  = new Mat();
 
     static {
```

Comment:
We got this from an OpenCv examples.
I think it's necessary to initialize the OpenCV before going through the Main function otherwise we end up with async loading and a black screen while waiting for the initialization.